CIP-0068/README.md

```diff
@@ -331,25 +330,53 @@ We want to bring the metadata of the RFT `d5e6bf0500378d4f0da4e8dde6becec7621cd8
 
 In order to reference the correct UTxO containing the metadata, it needs to be authenticated, otherwise metadata spoofing attacks become possible. One way to achieve that is by adding an NFT (`reference NFT`) to the UTxO. This NFT needs to under the same Policy ID as the `user token`, followed by an asset name pattern defined in the standard. This way you create a secure link between `reference NFT` and `user token` without the need for any extra data and you can make use of this off-chain and on-chain. 
 
 The security for the link is derived from the minting policy itself, so it's important to write the validator with the right constraints and rules since this CIP solely defines the interface to keep flexibility as high as possible.
 
-
 ### Backward Compatibility
 
 To keep metadata compatibility with changes coming in the future, we introduce a `version` field in the datum.
 
+### Extending & Modifying this CIP
+
+> The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+> NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+> "OPTIONAL" in this section are to be interpreted as described in
+> [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+In order to prevent conflicting updates in the future; the addition of new asset classes following, or as part of, this
+standard MUST be submitted as a new CIP providing their own justification, implementation, rationale, and community
+review prior to official acceptance. Newly proposed `asset_name_labels` SHOULD NOT be added to 
+[CIP-0067](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0067) until the accompanying CIP has matured 
+through the community review and feedback stage to a point that it is considered in the `Under Review` status and is
+assigned a tentative CIP number by the CIP Editors panel.
+
+A brief reference to new asset classes MAY be added to this document after the accompanying CIP achieves the `accepted`
+status. Documentation describing these token asset classes MUST be fully encapsulated within their individual CIPs and a
+link MUST be provided to that CIP within this document.
+
+If a modification or change is deemed necessary to one of the asset classes contained within this document, namely Asset
+Name Labels: 100, 222, 333, or 444; which do not fundamentally change the nature, use, or reference of the tokens MAY
+be made as a modification of this document. However, any change proposed that presents a non-backwards compatible change
+MUST include an accompanying `version` field iteration and both specifications for the proposed, current, and historical
+versions of the format MUST be maintained in a logical way to assist future implementors who may encounter a version of
+these tokens from any point in time.
+
```

Comment:
```suggestion
Likewise, when such modifications or changes are made to the newer asset classes defined in separate CIPs, those CIPs MUST also be updated to similarly iterate their own `version` field and include appropriate documentation for all possible versions.
```

I think this would be implied in most cases, but the versioning system as a whole (for all material under the CIP-0068 umbrella) only has a maintainable `version` field when we require that it be done for _all_ compatible token types.

What I _don't_ know (not being an implementor): if this would require CIP updates to _all_ tokens classes in the registry every time there was a version bump on one of the cardinal 4 token types.  Perhaps @Crypto2099 that question could be anticipated & addressed here as well?

Reply:
In spite of our agreement @Crypto2099 at the meting today, to require "boilerplate" language in each extending CIP (https://github.com/cardano-foundation/CIPs/pull/551#issuecomment-1707400077), it might actually be better ensure in the CIP-0068 that all extending CIPs inherently use their `version` in the same way.  This might make implementations more resistant to authors' or editor's omissions, and allow extension authors to focus on the parts of their CIPs that are _unique_ 🤔

Reply:
As @Crypto2099 is explaining in the CIP meeting right now, this appears to be resolved by https://github.com/cardano-foundation/CIPs/pull/586/commits/1ec10c593293849977245e81bf0358885031b267.